pkg/config/load.go

```diff
@@ -15,6 +15,23 @@ func (c *Config) LoadFromEnv() {
 	v := reflect.ValueOf(c).Elem()
 	for i := 0; i < v.Type().NumField(); i++ {
 		f := v.Type().Field(i)
+		if defaultValue, ok := f.Tag.Lookup(DefaultTag); ok {
```

Comment:
We actually have a couple areas where we do if-nil-hard-coded-default. I like this better and I'll have a follow-up PR to update those.